plugins/api-docs/src/components/ApisCards/ProvidedApisCard.test.tsx

```diff
@@ -74,7 +74,7 @@ describe('<ProvidedApisCard />', () => {
     );
 
     expect(getByText(/Provided APIs/i)).toBeInTheDocument();
-    expect(getByText(/No component provides this API/i)).toBeInTheDocument();
+    expect(getByText(/does not provide its own APIs/i)).toBeInTheDocument();
```

Comment:
"its own" vs "any" in the above consumed case, let's be consistent. I don't think that the API has to be owned by the component, you can also implement APIs provided by others.

Reply:
I think for me, if I _own_ a microservice and that service _provides_ an API, it's the API _for_ the microservice.  I might provide say only one API - my api - but I might consume 12 different APIs from other teams. Authentication APIs, user profile APIs, observability APIs, etc.  So I added "its" as possessive specifically to help enforce that it's an API fro\m this thing, from this component/this microservice, which is vastly different than just "it uses this other thing".

And then when you're in a system, a system might have 12 microservices, each one might then provide it's own APIs.  So the system as a whole may be a collection of 18 different APIs, but I interpret that the system has a collection of APIs, but it's not OWNED by the system. To take a real example, I might have a dozen squads own a dozen services, and each of them provide "their own" api, but then an architecture team or a product manager might own "the system" which has those APIs, and that's why I'm trying to veer away from saying "apis owned by a system" but rather APIs which are IN a system.

So I'll be honest, I really like this wording but I've not convinced you, have I? 😁

Reply:
I think there are a lot of use cases where you provide an implementation for an API but are not owning it - I would even say it's the majority of cases. For example the format of a metric endpoint like prometheus. It's an API, and the API is defined by the consumer. Most of the time this isn't really visible and probably also not tracked in Backstage.

As it leads to confusion, I would explicitly avoid the term of "owning" here. I would reserve that for team ownership.

Reply:
Ok.  Let me let my head clear a bit today, get to some other things and I'll take a fresh look, sort of step back and see if I understand at a broad stroke before taking another swag of wording.  Appreciate all the feedback!

I'll say I took a few of one of my team's existing Swagger definitions (defined statically in GitHub today along side static Markdown tables that link to them!)) and threw them in, and immediately these types of tables are really exposing what we have in a much more digestible way hence my wanting to dig into them further. So all great value add to an org to understand their engineering catalog.